vNext/src/Microsoft.AspNet.OData/Routing/Conventions/DefaultODataRoutingConvention.cs

```diff
@@ -43,13 +43,36 @@ public ActionDescriptor SelectAction(RouteContext routeContext)
                     controllerName = entitySetSegment.EntitySet.Name;
                 }
 
+                var operationImportSegment = odataPath.FirstSegment as OperationImportSegment;
```

Comment:
Do you want to create a new class in Routing\Convention and declaring a new interface for it (same interface as IODataRoutingConvention)? I believe this should be done in a different branch since it will involve registering new routing convention and handling it in code. 

Currently, the only function in place is used in the Customer context, so I have routed the function to the Customer controller. Do you envision a separate controller for function/action imports?